### DISCUSSION TITLE
Options

### DISCUSSION THREAD

**Discussion by Tptuaazn** [BG-Software-LLC/SuperiorSkyblock2#237] [Implemented Feature Requests] [answered]
A lot of players don't want to use this feature. The simply not adding permissions allows that, but they don't seem to understand.
So I came up with the idea of ​​creating a yaml that makes it easier for people to disable it.

Exam:

Bank: true

Missions: false

**Comment by OmerBenGera** [comment, marked as answer]
This is gonna be implemented soon :)
